<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UWBMap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMap">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="PozyxMap" name="pozyx_map" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGraphicsView" name="map">
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="coordinates">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMQTTLog">
       <attribute name="title">
        <string>MQTT Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTextBrowser" name="textMQTTLog">
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAgentMessages">
       <attribute name="title">
        <string>Agent Messages</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextBrowser" name="textAgentMessages">
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAgents">
       <attribute name="title">
        <string>Agents</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="layoutTabAgents"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>MQTT Connection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="QLabel" name="MQTTStatus">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="MQTTReset">
               <property name="text">
                <string>Reconnect</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Local MQTT Connection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="localMQTTStatus">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="localMQTTReset">
               <property name="text">
                <string>Reconnect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Agent Connection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="javaAgentStatus">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="javaAgentReset">
               <property name="text">
                <string>Reconnect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonResendCharging">
            <property name="text">
             <string>Resend Charging Stations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemoveCharging">
            <property name="text">
             <string>Remove Charging Stations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonReset">
            <property name="text">
             <string>Reset Map</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="table_similarity_measure">
       <attribute name="title">
        <string>Similarity Measure</string>
       </attribute>
       <widget class="QTableWidget" name="table_widget_similarity_measure">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>40</y>
          <width>801</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="button_results">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>10</y>
          <width>201</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_file1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>201</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Physical Agent File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_file2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>201</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Digital Agent File</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionMQTT_logs">
   <property name="text">
    <string>MQTT logs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PozyxMap</class>
   <extends>QWidget</extends>
   <header>uwbmap.mainwindow.map</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
